content/hi/docs/concepts/architecture/nodes.md

```diff
@@ -0,0 +1,649 @@
+
+```yaml
+shutdownGracePeriodByPodPriority:
+  - priority: 100000
+    shutdownGracePeriodSeconds: 10
+  - priority: 10000
+    shutdownGracePeriodSeconds: 180
+  - priority: 1000
+    shutdownGracePeriodSeconds: 120
+  - priority: 0
+    shutdownGracePeriodSeconds: 60
+```
+
+उपरोक्त तालिका का अर्थ है कि `प्राथमिकता` मूल्य> = 100000 वाले किसी भी फली को मिलेगा
+रुकने के लिए सिर्फ 10 सेकंड, मान> = 10000 और <100000 वाले किसी भी पॉड को 180 मिलेगा
+सेकंड रुकने के लिए, >= 1000 और <10000 के मान वाले किसी भी पॉड को रुकने के लिए 120 सेकंड मिलेंगे।
+अंत में, अन्य सभी पॉड्स को रुकने के लिए 60 सेकंड मिलेंगे।
+
+किसी को सभी वर्गों के अनुरूप मान निर्दिष्ट करने की आवश्यकता नहीं है। के लिए
+उदाहरण के लिए, आप इसके बजाय इन सेटिंग का उपयोग कर सकते हैं:
+
+|Pod priority class value|Shutdown period|
+|------------------------|---------------|
+| 100000                 |300 seconds    |
+| 1000                   |120 seconds    |
+| 0                      |60 seconds     |
+
+
+उपरोक्त मामले में, `कस्टम-क्लास-बी` वाले पॉड उसी बकेट में जाएंगे
+शटडाउन के लिए `कस्टम-क्लास-सी` के रूप में।
+
+यदि किसी विशेष श्रेणी में पॉड्स नहीं हैं, तो क्यूबलेट प्रतीक्षा नहीं करता है
+उस प्राथमिकता सीमा में पॉड्स के लिए। इसके बजाय, क्यूबलेट तुरंत छोड़ देता है
+अगली प्राथमिकता वर्ग मूल्य सीमा।
+
+यदि यह सुविधा सक्षम है और कोई कॉन्फ़िगरेशन प्रदान नहीं किया गया है, तो कोई ऑर्डरिंग नहीं है
+कार्रवाई की जाएगी।
+
+इस सुविधा का उपयोग करने के लिए `GracefulNodeShutdownBasedOnPodPriority` को सक्षम करना आवश्यक है
+[फीचर गेट](/docs/reference/command-line-tools-reference/feature-gates/)
+, और इसमें `शटडाउनग्रेसपीरियोडबायपॉडप्रायोरिटी` सेट करना
+[क्यूबलेट कॉन्फ़िगरेशन](/docs/reference/config-api/kubelet-config.v1beta1/)
+पॉड प्राथमिकता वर्ग मान वाले वांछित कॉन्फ़िगरेशन के लिए और
+उनकी संबंधित शटडाउन अवधि।
+
+{{< नोट >}}
+ग्रेसफुल नोड शटडाउन के दौरान पॉड की प्राथमिकता को ध्यान में रखने की क्षमता पेश की गई थी
+कुबेरनेट्स v1.23 में एक अल्फा फीचर के रूप में। कुबेरनेट्स {{< skew currentVersion >}}
+सुविधा बीटा है और डिफ़ॉल्ट रूप से सक्षम है।
+{{< /नोट >}}
+
+मेट्रिक्स `ग्रेसफुल_शटडाउन_स्टार्ट_टाइम_सेकंड्स` और `ग्रेसफुल_शटडाउन_एंड_टाइम_सेकंड्स`
+नोड शटडाउन की निगरानी के लिए क्यूबलेट सबसिस्टम के तहत उत्सर्जित होते हैं।
+
+## नॉन ग्रेसफुल नोड शटडाउन {#नॉन-ग्रेसफुल-नोड-शटडाउन}
+
+{{< feature-state state="beta" for_k8s_version="v1.26" >}}
+
+क्यूबलेट के नोड शटडाउन मैनेजर द्वारा नोड शटडाउन कार्रवाई का पता नहीं लगाया जा सकता है,
+या तो क्योंकि कमांड द्वारा उपयोग किए जाने वाले इनहिबिटर लॉक मैकेनिज्म को ट्रिगर नहीं करता है
+क्यूबलेट या उपयोगकर्ता त्रुटि के कारण, यानी शटडाउनग्रेसपीरियोड और
+ShutdownGracePeriodCriticalPods ठीक से कॉन्फ़िगर नहीं किए गए हैं। कृपया ऊपर देखें
+अनुभाग [ग्रेसफुल नोड शटडाउन](#ग्रेसफुल-नोड-शटडाउन) अधिक विवरण के लिए।
+
+जब एक नोड बंद हो जाता है लेकिन क्यूबलेट के नोड शटडाउन मैनेजर द्वारा पता नहीं लगाया जाता है, तो pods
+जो कि {{< glosary_tooltip text="StatefulSet" term_id="statefulset" >}} का हिस्सा हैं, समाप्त होने की स्थिति में अटक जाएंगे
```

Comment:
Typo in the `glossary_tooltip` shortcode name. Also, localize the "text" parameter passed to the shortcode to Hindi.

```suggestion
जो कि {{< glosary_tooltip text="स्टेटफुलसेट" term_id="statefulset" >}} का हिस्सा हैं, समाप्त होने की स्थिति में अटक जाएंगे
```